presslabs/wordpress-composer-builder
# Use .netrc to do github pushes

Let drone publish the patched Wordpress, so we don't need to manually run the make scripts.

We do that by using the `.netrc` config via the `setup-github-credentials.sh` script, that is inspired by the `kluster-toolbox` way of doing this kind of operations.

##### Tests:
- make a new branch from `fix-drone`
- remove 
```
when:
      event: push
      branch: master
```

- add a dummy commit
- push
- it **should** start a new Drone job that will succeed and will push new commits to https://github.com/presslabs/wordpress
